solidity/contracts/igps/InterchainGasPaymaster.sol

```diff
@@ -23,6 +23,9 @@ contract InterchainGasPaymaster is
     /// @notice Keyed by remote domain, the gas oracle to use for the domain.
     mapping(uint32 => IGasOracle) public gasOracles;
 
+    /// @notice The benficiary that can receive native tokens paid into this contract.
+    address public beneficiary;
```

Comment:
Just double checking that we know that this storage addition is safe

Reply:
yep purely additive

Reply:
prior to performing the contract upgrade I'll be sure to inspect the storage layout before & what it'll look like after to ensure it's only additive https://book.getfoundry.sh/reference/forge/forge-inspect?highlight=storage%20layout#examples

Reply:
Feel free to do https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/1742 if you want :P